src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```diff
@@ -79,6 +85,74 @@ _parse_cli_pairs() {
   done
 }
 
+_set_backend_defaults() {
+  case "${dynamo_args["backend"]}" in
+    vllm)
+      :
+      ;;
+    sglang)
+      dynamo_args["prefill-cmd"]="python3 -m dynamo.sglang.worker"
```

Comment:
No need to do this here. We can already have it flow from cmd_args.dynamo section of toml file.